ichnaea/webapp/app.py

```diff
@@ -66,4 +73,9 @@ def log_response(status, headers, exc_info=None):
 
 
 if __name__ == "__main__":
-    serve(log_access_factory(main(ping_connections=True)), host="0.0.0.0", port=8000)
+    if "--check" in sys.argv:
+        main(ping_connections=False)
```

Comment:
This is a pretty benign mode, however it makes sure that the webapp loads and it'll suss out most configuration issues. I say "most" because configuration throws errors "on demand" and this app doesn't log configuration, so there are a couple of configuration variables that throw an error during the http request/response time. However, that will send an error to sentry and it's not at startup, so that's "ok".